src/components/Menu/Menu.tsx

```diff
@@ -1,36 +1,38 @@
-import { Box } from "@mui/system";
-import Icon from "@mui/material/Icon";
-import Typography from "@mui/material/Typography";
-
 import { ReactComponent as MySvgFile } from "../../assets/logos/menuLogo.svg";
+import MenuNavItem from "../MenuNavItem";
+
+import { ADDITIONAL_NAV_INFO, NAV_INFO } from "./helpers";
+import { LogoIcon, LogoText, MenuBox, MenuLogoBox, StyledLine } from "./styled";
 
 const Menu = () => {
   return (
-    <Box sx={{ backgroundColor: "#363740", width: "255px", height: "100vh" }}>
-      <Box
-        sx={{
-          display: "flex",
-          alignItems: "center",
-          gap: "12px",
-          margin: "41px 0 0 32px",
-        }}
-      >
-        <Icon sx={{ width: "32px", height: "32px" }}>
+    <MenuBox>
+      <MenuLogoBox>
+        <LogoIcon>
           <MySvgFile />
-        </Icon>
-        <Typography
-          variant="subtitle1"
-          sx={{
-            fontSize: "19px",
-            fontWeight: 700,
-            color: "#A4A6B3",
-            letterSpacing: "0.4px",
-          }}
-        >
-          Dashboard Kit
-        </Typography>
-      </Box>
-    </Box>
+        </LogoIcon>
+        <LogoText>Dashboard Kit</LogoText>
+      </MenuLogoBox>
+      {NAV_INFO.map((item) => (
+        <MenuNavItem
+          key={item.text}
+          to={item.to}
```

Comment:
fields in `item` and expected props in `MenuNavItem` the same. So we can use `{...item}` instead of passing each prop separately

Reply:
Fixed